From: Simon McVittie <smcv@debian.org>
Date: Fri, 20 Mar 2020 13:18:50 +0000
Subject: bzr: Run brz in preference to bzr

Bazaar-NG (bzr) relies on Python 2, which has reached end-of-life, and
appears to be essentially unmaintained itself. Try using Breezy (brz),
a "friendly fork" of bzr that has been ported to Python 3 and is
maintained.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://github.com/flatpak/flatpak-builder/pull/333
---
 src/builder-source-bzr.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/builder-source-bzr.c b/src/builder-source-bzr.c
index 56de6e7..ceeec94 100644
--- a/src/builder-source-bzr.c
+++ b/src/builder-source-bzr.c
@@ -120,11 +120,14 @@ bzr (GFile   *dir,
      GError **error,
      ...)
 {
+  g_autofree char *brz = NULL;
   gboolean res;
   va_list ap;
 
+  brz = g_find_program_in_path ("brz");
+
   va_start (ap, error);
-  res = flatpak_spawn (dir, output, 0, error, "bzr", ap);
+  res = flatpak_spawn (dir, output, 0, error, brz ? brz : "bzr", ap);
   va_end (ap);
 
   return res;
